third_party/2and3/pymysql/cursors.pyi

```diff
@@ -17,34 +18,35 @@ class Cursor:
     def __init__(self, connection: Connection) -> None: ...
     def __del__(self) -> None: ...
     def close(self) -> None: ...
-    def setinputsizes(self, *args): ...
-    def setoutputsizes(self, *args): ...
-    def nextset(self): ...
-    def execute(self, query: str, args: Optional[Any] = ...) -> int: ...
-    def executemany(self, query: str, args) -> int: ...
-    def callproc(self, procname, args=...): ...
+    def setinputsizes(self, *args) -> None: ...
+    def setoutputsizes(self, *args) -> None: ...
+    def nextset(self) -> Optional[bool]: ...
+    def mogrify(self, query: Text, args: QueryArgs = ...) -> Text: ...
```

Comment:
This actually calls `_ensure_bytes()` on the query and args, so it always returns `str`, even in Python 2.

Reply:
What is the advantage of using `str` instead of `Text` for py2and3 codebase? I think `Text` is better in this case.

Reply:
`Text` will be `str` in py3 and `unicode` in py2, while in this case, the type is `str` in both versions. So `Text` gives the wrong type in py2

Reply:
Changed to str